Leaf API.postman_collection.json

```diff
@@ -1421,6 +1026,631 @@
+														"type": "text/javascript"
+													}
+												}
+											],
+											"request": {
+												"method": "POST",
+												"header": [
+													{
+														"key": "Authorization",
+														"value": "Bearer {{leaf_token}}",
+														"type": "text"
+													},
+													{
+														"key": "Content-Type",
+														"value": "application/json",
+														"type": "text"
+													}
+												],
+												"body": {
+													"mode": "raw",
+													"raw": "{\n    \"applicationName\": \"{{trimble_application_name}}\",\n    \"serviceIdentityId\": \"{{trimble_service_identity_id}}\",\n    \"userName\": \"{{trimble_username}}\",\n    \"password\": \"{{trimble_password}}\"\n}",
+													"options": {
+														"raw": {
+															"language": "json"
+														}
+													}
+												},
+												"url": {
+													"raw": "https://{{leaf_api_url}}/services/usermanagement/api/users/{{leaf_user_id}}/trimble-credentials",
+													"protocol": "https",
+													"host": [
+														"{{leaf_api_url}}"
+													],
+													"path": [
+														"services",
+														"usermanagement",
+														"api",
+														"users",
+														"{{leaf_user_id}}",
+														"trimble-credentials"
+													]
+												}
+											},
+											"response": []
+										},
+										{
+											"name": "Delete credential of LeafUser",
+											"event": [
+												{
+													"listen": "test",
+													"script": {
+														"exec": [
+															"pm.test(\"Successful DELETE request\", function () {",
+															"    pm.expect(pm.response.code).to.be.oneOf([204,404]);",
+															"});"
+														],
+														"type": "text/javascript"
+													}
+												}
+											],
+											"request": {
+												"method": "DELETE",
+												"header": [
+													{
+														"key": "Authorization",
+														"value": "Bearer {{leaf_token}}",
+														"type": "text"
+													},
+													{
+														"key": "Content-Type",
+														"value": "application/json",
+														"type": "text"
+													}
+												],
+												"url": {
+													"raw": "https://{{leaf_api_url}}/services/usermanagement/api/users/{{leaf_user_id}}/trimble-credentials",
+													"protocol": "https",
+													"host": [
+														"{{leaf_api_url}}"
+													],
+													"path": [
+														"services",
+														"usermanagement",
+														"api",
+														"users",
+														"{{leaf_user_id}}",
+														"trimble-credentials"
+													]
+												}
+											},
+											"response": []
+										}
+									]
+								},
+								{
+									"name": "CNHi",
+									"item": [
+										{
+											"name": "Get credential of LeafUser",
```

Comment:
Lest include the provider name in the title of this endpoint

Reply:
Added it: https://github.com/Leaf-Agriculture/Leaf-API-Postman-Collection/pull/44/commits/8194d7cb4685389a79c930a74168de128fbfb605